hassio/snapshots/hassio-snapshots.html

```diff
@@ -63,13 +63,18 @@
                 </paper-checkbox>
               </template>
             </div>
-          </div>
-          <div class='card-actions'>
-            <paper-button disabled='[[creatingSnapshot]]' on-click='createSnapshot'>Create</paper-button>
+            <div>
+              Password:
+              <paper-checkbox checked='{{snapshotHasPassword}}'>Password protected</paper-checkbox>
+              <paper-input hidden=[[!snapshotHasPassword]] autofocus label='Password' type='password' value='{{snapshotPassword}}'></paper-input>
```

Comment:
Please add quotes around attribute values (the one for hidden). Also, does hidden work ? You're not adding it as a class ?

Reply:
hidden works

Reply:
That's weird since it's not part of the paper-input API: https://www.webcomponents.org/element/PolymerElements/paper-input/elements/paper-input

And thus it means it's very fragile and can easily break in the future. Your call what you want to do.